docs/example-scenario/apps/devops-dotnet-webapp.md

```diff
@@ -112,7 +112,7 @@ For general guidance on designing secure solutions, see the [Azure Security Docu
 
 Review the [typical design patterns for resiliency][design-patterns-resiliency] and consider implementing these where appropriate.
 
-You can find a number of [resiliency recommended practices for App Service][resiliency-app-service] on the architecture center.
+You can find a number of [recommended practices for App Service][resiliency-app-service] on the architecture center.
```

Comment:
We should use "in the Azure Architecture Center" here.

Reply:
updated.